tst/testbugfix/00094.tst

```diff
@@ -0,0 +1,6 @@
+# 2005/08/23 (FL)
+# commented out the test and the error message,
+# since a different message is printed on 32 bit systems and 64 bit systems
+gap> a := 2^(8*GAPInfo.BytesPerVariable-4)-1;;
+gap> Unbind( x );
+gap> # x := [-a..a];;
```

Comment:
We now have directories for 32/64 bit specific tests, so I guess this should be copied to the two, with the last line activated